python_modules/dagster/dagster/_core/execution/backfill.py

```diff
@@ -127,11 +127,15 @@ def is_valid_serialization(self, workspace: IWorkspace) -> bool:
         else:
             return True
 
-    def get_partitions_status_counts_and_totals_by_asset(
+    def get_partitions_status_counts_and_totals_per_asset(
         self, workspace: IWorkspace
-    ) -> Mapping[AssetKey, Tuple[Mapping[BackfillPartitionsStatus, int], int]]:
+    ) -> Sequence[Tuple[AssetKey, Mapping[BackfillPartitionsStatus, int], int]]:
```

Comment:
In Python 3, dictionaries are actually ordered, so I don't think we need to switch this to a sequence.

Reply:
ooo this is cool